src/shared/globalFileNames.ts

```diff
@@ -4,6 +4,7 @@ export const GlobalFileNames = {
 	glamaModels: "glama_models.json",
 	openRouterModels: "openrouter_models.json",
 	requestyModels: "requesty_models.json",
-	mcpSettings: "cline_mcp_settings.json",
+	mcpSettings: "mcp_settings.json",
```

Comment:
Changing this seems like it might cause problems without a migration plan?

Reply:
I mentioned this in the "Considerations" section.  What migration plan would you recommend?